tests/core/test_model_manager.py

```diff
@@ -616,6 +618,60 @@ def test_find_without_on_disk_model(good_model_path, reset_globals):
         assert not os.path.exists("some_pretend_model")
 
 
+def test_find_without_on_disk_model_and_load_path(good_model_path, reset_globals):
+    """Make sure that ephemeral models can be found without existing on disk
+    when a shared load_path is configured
+    """
+    with tempfile.TemporaryDirectory() as workdir:
+        with temp_config(
+            {
+                "load_path": workdir,
+                "model_management": {
+                    "finders": {
+                        "default": {
+                            "type": TestFinder.name,
+                            "config": {
+                                "fallback_to_local": False,
```

Comment:
Why is this `fallback_to_local` config required?

It looks like that is used only if the model path actually exists, but for this test the path should not exist

```
        if os.path.exists(model_path):
            if self._fallback_to_local:
...
```